src/middleware.js

```diff
@@ -23,7 +23,7 @@ export const createOfflineMiddleware = (config: Config) => (store: any) => (
 
   // create promise to return on enqueue offline action
   if (action.meta && action.meta.offline) {
-    promise = registerAction(offlineAction.meta.transaction);
+    promise = registerAction(state.offline.lastTransaction);
```

Comment:
I would use `offline` instead of `state.offline`. Just to make sure we are accessing the offline state using `offlineStateLens()`.

Reply:
whoops